sentry/src/main/java/io/sentry/NoOpHub.java

```diff
@@ -7,7 +7,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-final class NoOpHub implements IHub {
+public final class NoOpHub implements IHub {
```

Comment:
you need to annotate this class with `@Open` too

Reply:
It stays final, why would it need to be `@Open`?

Reply:
ups my bad, CI failed and I misread the issue, they need to run `./gradlew apiDump` because there's a new public class in our public API